testing/ember-app/tests/utils/function/rendering-test.ts

```diff
@@ -81,4 +81,63 @@ module('Utils | trackedFunction | rendering', function (hooks) {
     await settled();
     assert.dom('out').hasText('4');
   });
+
+  test('it can be reexecuted', async function (assert) {
+    const store = {
+      index: 0,
+      data: [[{ id: 1 }], [{ id: 1 }, { id: 2 }], [{ id: 1 }]],
+      fetch(page: Number) {
+        // For some reason the resource prop is triggered again
+        // after the refetch call which causes the array index to be out of bound.
+        // Im not sure if something is not working correctly here
+        // or if i just was overlookging a property trigger somewhere.
+        const value =
```

Comment:
@NullVoxPopuli I implemented a test as you suggested  it. Im running into a problem here tough. After i trigger the re-execution of the `trackedFunction` with `click`, the resource property is, as expected, evaluated again. But then the resource property is triggered one last time without the args changing or the execution function being called. Do you maybe know why this happens? This is why i had to use a index  check here so were not running into a out-of-bound.